node/README.md

```diff
@@ -0,0 +1,48 @@
+# AnyCloud
+
+AnyCloud is a Lambda alternative for Node.js that works with multiple cloud providers. We’re building the features of Lambda while fixing the problems we have run into while using it. Namely:
+
+- Vendor lock-in
+- Cold starts
+- Limited runtime (10-15m)
+- Stateless
+- Cumbersome to run locally
+- Unintuitive to version and remove inactive functions
+- HTTPS support not included
+
+## Documentation
+
+Visit our [docs](https://alantech.gitbook.io/anycloud)
+
+## Status
+- [ ] Public Alpha: Anyone can sign up, but go easy on us
+- [ ] Public Beta: Stable enough for most non-enterprise use-cases
+- [ ] Public: Production-ready
+
+## Installation
+
+Simply `npm i -g anycloud` to get access to the `anycloud` cli application.
+
+## Usage
+
+TODO
```

Comment:
we can probably go by with just this and not repeat information from the top level README?

Reply:
Unfortunately no, [the npm package has no README](https://www.npmjs.com/package/anycloud) and we'll probably need to do similar things for other language packages, too. Like I think Python prefers restructured text, so we'll need a .rst version of the README when we write that library?

Reply:
ah, got it. that is unfortunate

Reply:
honestly it might be possible to have a template at the top level dir and then as part of CI it generates the appropriate file. We can even use markdown since things like <https://github.com/miyakogi/m2r> exist